clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/RetirementMessage.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.common.internal.messages;
+
+import org.ehcache.clustered.common.internal.store.Chain;
+
+import java.util.UUID;
+
+/**
+ * This message is sent by the Active Entity to Passive Entity.
+ */
+public class RetirementMessage extends EhcacheEntityMessage {
```

Comment:
I do not understand why these messages are named `RetirementMessage`. That's how they are used not what they convey from the active to the passive.